src/routes.js

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 import Lesson from './components/Lesson/Lesson';
 import getRouteObject from './routeObject';
+import store from './store';
 
+function validPathTest(course, lesson, path){
+  const state = store.getState().lessons; 
+
+  if(!lesson){
+    for(var key in state){
+      if(course == state[key]['course']){
```

Comment:
Du skal nesten alltid bruke `===` i stedet for `==`. Eneste unntaket er hvis du skal sjekke om noe er null eller undefined samtidig, da kan du bruke `== null`.